app/views/helpers/lists.php

```diff
@@ -305,36 +305,38 @@ public function displayVisibilityOption($listId, $value)
         <?php
     }
 
-    public function displayIsEditableByAnyOption($listId, $isChecked)
+    /**
+     * Display editable_by options for sentence lists.
+     *
+     * @param int    $listId List ID.
+     * @param string $value  Currently set option.
+     */
+    public function displayEditableByOptions($listId, $value)
     {
         $this->Javascript->link(
-            JS_PATH . 'sentences_lists.set_option.js', false
+            JS_PATH.'sentences_lists.set_option.js', false
         );
         ?>
         <dl>
             <?php
-            echo $this->Html->tag('dt', __('Permission to edit', true));
-            if ($isChecked) {
-                $checkboxValue = 'checked';
-            } else {
-                $checkboxValue = '';
-            }
-
-            echo "<div class='is-editable loader-container'></div>";
+                $title = __('Who can add/remove sentences', true);
+                $loader = "<div class='is-editable loader-container'></div>";
+                echo $this->Html->tag('dt', $title.$loader);
 
-            echo $this->Form->checkbox(
-                'isEditableByAnyone',
-                array(
-                    "id" => "editableCheckbox",
-                    "name" => "isEditableByAnyone",
-                    "checked" => $checkboxValue,
-                    "data-list-id" => $listId
-                )
-            );
-            echo $this->Html->tag('label',
-                __('Allow anyone to add and delete sentences from the list', true),
-                array('for' => 'editableCheckbox')
-            );
+                echo $this->Form->radio(
+                    'editable_by',
+                    array(
+                        'anyone' => __('Anyone', true),
+                        'creator' => __('Only Me', true),
+                        'no_one' => __('No One', true),
```

Comment:
Small detail: only the first word should be capitalized.
* `Only Me` => `Only me`
* `No One` => `No one`